src/main/kotlin/racingcar/Race.kt

```diff
@@ -0,0 +1,36 @@
+package racingcar
+
+fun main() {
+    val inputView = InputView()
+    val outputView = OutputView()
+
+    inputView.drawNumberOfCarInput()
+    val numberOfCar = readLine()!!.toInt()
+    inputView.drawNumberOfTryInput()
+    val numberOfTry = readLine()!!.toInt()
+
+    val race = Race(numberOfCar)
+
+    for (i in 1..numberOfTry) {
+        race.process()
+        outputView.drawRace(race)
+    }
+}
+
+class Race(numberOfCar: Int) {
+    val carList: MutableList<Car> = mutableListOf()
+
+    init {
+        for (i in 1..numberOfCar) {
+            carList.add(Car(0))
+        }
+    }
```

Comment:
```suggestion
class Race(numberOfCar: Int) {
    val carList: List<Car> = List(numberOfCar) {
        Car(0)
    }
```

Race에 대해 초기화하는 로직 마음에 들어요! 잘하셨어요! 여기서 살짝 포인트를 잡는다면, List 함수를 통하여 가변인 부분을 불변으로 위와 같이 지킬 수가 있습니다.